docs/shared/guides/using-tailwind-css-in-react.md

```diff
@@ -50,6 +50,8 @@ npx tailwindcss init -p
 
 This creates the required files with a general boilerplate implementation.
 
+**Note**: For package based monorepos your apps are located in `packages/*`
```

Comment:
Instead of adding a note here, why don't you just change the `shell` snippet above to this:

```
```shell
cd {path to your app}
npx tailwindcss init -p
```
```